crowdsec-docs/unversioned/getting_started/next_steps.md

```diff
@@ -0,0 +1,42 @@
+---
+id: next_steps
+title: Post Installation Steps
+---
+
+Now that you have installed CrowdSec, we have a post installation steps to help continue your journey.
```

Comment:
I suggest a middle ground with what @rr404 suggested : We have a separate "pre flight checklist" in the user guides, and we reference it here. Hopefully, in 90% of the case, people won't have any issues, so let's not bother them "by default" ?